Dockerfile

```diff
@@ -0,0 +1,72 @@
+FROM  ubuntu:20.04 as builder
+
+USER  root
+
+# ideally the repo would only have one code tree with versions controlled by tags and support branches
+# due to this not being the case using quay.io (or similar) to automatically build the tagged version
+# would be dangerous as this value MUST be manually updated to pick up the correct code
+ARG VER_FOLDER="v1.2.3"
+ARG VER_MINIMAP2="2.17"
+
+ENV OPT=/opt/xmatchview
+# have to split as OPT needs to exist to use in the next ENV
+ENV PATH=$OPT/bin:$PATH
+
+RUN apt-get -yq update
+RUN apt-get install -yq --no-install-recommends curl ca-certificates
+
+WORKDIR /install_tmp
+
+RUN mkdir -p $OPT/bin
+
+RUN curl --retry 10 -o distro.tar.bz2 -sSL https://github.com/lh3/minimap2/releases/download/v${VER_MINIMAP2}/minimap2-${VER_MINIMAP2}_x64-linux.tar.bz2 \
+&& mkdir -p distro \
+&& tar -C distro --strip-components 1 -xjf distro.tar.bz2 \
+&& cp distro/minimap2 $OPT/bin/.
+
+COPY tarballs/fonts $OPT/fonts
+RUN chmod +r -R $OPT/fonts
+
+# add tests to OPT bin
+RUN mkdir -p $OPT/tests
+COPY $VER_FOLDER/test $OPT/tests/test
+COPY $VER_FOLDER/test-hive $OPT/tests/test-hive
+RUN chmod +r -R $OPT/tests
+RUN chmod +x $OPT/tests/test/*.sh
+
+# add the scripts to the OPT bin
+COPY $VER_FOLDER/*.py $OPT/bin/
+COPY $VER_FOLDER/*.sh $OPT/bin/
+RUN chmod +rx $OPT/bin/*.sh $OPT/bin/*.py
+
+FROM  ubuntu:20.04
+
+# LABEL maintainer="???"\
+#       version="???" \
+#       description="xmatchview"
+
```

Comment:
Could you just delete the commented out code?

Reply:
I left that as an indicator that it should be completed as appropriate.  You don't really need the version line, but maintainer is a good place to put a shared mailbox email for support requests.

Reply:
Ok I see -- then can you add Rene Warren (rwarren@bcgsc.ca) as the maintainer?